frontend/src/components/profile/components/user-info/user-info.tsx

```diff
@@ -29,12 +37,36 @@ type Props = {
 const UserInfo: React.FC<Props> = ({ user, isDoctor, onEdit }) => {
   const birthdate = getFormattedDate(user.birthdate, DateFormat.D_MMMM_YYYY);
   const dispatch = useDispatch();
+  
+  const { handleSubmit, errors, control } = useForm<IProfession>({
+    defaultValues: DEFAULT_PROFESSION_VALUE,
+    mode: 'onChange',
+  });
+
+  const { professions } = useSelector(({ profile }: RootState) => ({
+    professions: profile.professions,
+  }));
+  
+  const professionOptions: IOption<string>[] = professions.map((profession: IProfession) => {
+    return {
+      label: profession[ProfessionKey.NAME],
+      value: profession[ProfessionKey.ID],
+    };
+  });
```

Comment:
Create helper `getProfessionOptions` and move it to the `frontend/src/components/profile/components/user-info/helpers/`